pkg/reconciler/v1alpha1/config/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// Package config holds the typed objects that define the schemas for
+// assorted ConfigMap objects on which the Route controller depends.
+package config
```

Comment:
There are several common config packages already, we should probably follow those: `pkg/autoscaler` has one, `pkg/logging` has another.